### PR TITLE
klientctl: allow to umount all machines at once

### PR DESCRIPTION
**Pull request by ppknap**
## Description
https://asciinema.org/a/47oocpuezxc2ax4euznfaqmqw

### CODE REVIEW

File: go/src/koding/klientctl/endpoint/machine/mount.go [hunk 1]
```diff
@@ -201,25 +206,80 @@ func (c *Client) Umount(options *UmountOptions) (err error) {
...
+	}
+
+	// Remove mounts.
+	for _, identifier := range identifiers {
+		umountReq := &machinegroup.UmountRequest{
+			Identifier: identifier,
+		}
+		var umountRes machinegroup.UmountResponse
+
+		if err := c.klient().Call("machine.umount", umountReq, &umountRes); err != nil {
```
Comment (rjeczalik) [inline go/src/koding/klientctl/endpoint/machine/mount.go]: What about making it best-effort attempt at unmounting all mounts? So if single unmount fails we keep going unmounting test of them.

**Review by rjeczalik — APPROVED** [review]

File: go/src/koding/klientctl/endpoint/machine/mount.go [hunk 2]
```diff
@@ -201,25 +206,80 @@ func (c *Client) Umount(options *UmountOptions) (err error) {
...
+		identifiers = mids
+	}
 
-	// Remove mount.
-	umountReq := &machinegroup.UmountRequest{
-		Identifier: options.Identifier,
+	// If there are no mounts we return nil.
+	if len(identifiers) == 0 {
+		fmt.Fprintf(os.Stdout, "There is nothing to unmount\n")
+		return nil
```
Comment (rjeczalik) [inline go/src/koding/klientctl/endpoint/machine/mount.go]: I think it should return non-nil error when len(options.Identifiers) == 1, just to be backward-compatible with previous behaviour.

**Comment by rjeczalik** [comment]
Rebased & resolved conflicts.

File: go/src/koding/klientctl/endpoint/machine/mount.go (hunk 2 above)
Comment (ppknap) [inline go/src/koding/klientctl/endpoint/machine/mount.go]: It is backward-compatibile, `len(options.Identifiers) == 1` is when user provides: "kd umount <mount>". The case above (`len(options.Identifiers) == 0`) covers the command "kd umount -a" when there is no mounts. Eg.  "kd umount --force" will result in error [here](https://github.com/koding/koding/pull/11096/files#diff-6c91a42efe84e0738dea5008055828c2R151).

**Review by cihangir — APPROVED** [review]

**Pull request merged**